docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
+
+To create a playlist from the server side:
+
+1. Create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can create playlist with Metadata also like below code.
```

Comment:
Kindly use colon at the end of the sentence as it is following by a code block. Also this sentence is not very clear, while reading. Can we edit it like "You can also create playlist with metadata using the following code:"

Please verify the technical accuracy for the sentence and share your opinion before updating.

Reply:
Done